app/javascript/components/change-logs/container.spec.js

```diff
@@ -0,0 +1,171 @@
+import { mountedComponent, screen } from "test-utils";
```

Comment:
Replace old file instead of replacing new file

Reply:
this one is old file.